### DISCUSSION TITLE
how do I move from Z-Wave JS to Z-Wave JS UI

### DISCUSSION THREAD

**Discussion by GHSTUDIO1** [zwave-js/zwave-js-ui#3451] [Q&A] [answered]
I am running Z-Wave JS with many automations.....I would like to move to Z-Wave JS UI but I don't want the names of my switches, etc to change.   How do I move the the UI version without having to change any of my automations?

**Comment by kpine** [comment, marked as answer]
I would recommend the Home Assistant [community forum](https://community.home-assistant.io/) as a better place for questions related to HA and its add-ons.

See this guide: https://community.home-assistant.io/t/switching-z-wave-js-addons-with-minimal-downtime-z-wave-js-official-to-z-wave-js-ui-community/409904
